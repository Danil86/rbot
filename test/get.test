do move to nearest reachable position block nearest block log then dig block nearest block log done