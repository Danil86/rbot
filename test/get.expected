["taskList",[[["move to",["nearest reachable position nearest block log"]],["dig",["nearest block log"]]]]]
